Rocket.Chat/fr.lproj/Localizable.strings

```diff
@@ -132,3 +154,8 @@
 "chat.messages.pinned.list.title" = "Messages épinglés";
 "chat.messages.starred.list.title" = "Messages favoris";
 
+// Settings
+"myaccount.settings.title" = "Settings";
+"myaccount.settings.contactus" = "Contact us";
+"myaccount.settings.license" = "License";
+"myaccount.settings.version" = "Version: %@ (%@)";
```

Comment:
@sowenjub I know it sucks, but could you help us translating this new strings? We're working to integrate with LingoHub soon.